main.py

```diff
@@ -0,0 +1,35 @@
+from time import sleep
+from sys import platform
+import signal
+
+from client.comm import Comm
+from modules.swarm_management.module.swarm_management import Module
+
+should_stop = False
+
+
+def main():
+    print("Starting application...\n")
+    module = Module(Comm())
+    print("Module created...")
+
+    while not should_stop:
+        module.process()
+        sleep(0.05)
+
+    module.stop()
+
+
+def stop(signal, frame):
+    global should_stop
+    should_stop = True
+
+
+signal.signal(signal.SIGINT, stop)
+signal.signal(signal.SIGTERM, stop)
+
+if platform != "win32":
```

Comment:
Is het niet handig om dit ook voor linux en mac te doen? of zijn dit de regels hierboven?

Reply:
komt uit de template dus ik ga ervan uit dat er overna gedacht is tbh